templates/watcher/config/watcher-decision-engine-config.json

```diff
@@ -0,0 +1,32 @@
+{
+    "command": "/usr/bin/watcher-decision-engine --config-dir /etc/watcher/watcher.conf.d",
+    "config_files": [
+      {
+        "source": "/var/lib/config-data/default/00-default.conf",
+        "dest": "/etc/watcher/watcher.conf.d/00-default.conf",
+        "owner": "watcher",
+        "perm": "0600"
+      },
+      {
+        "source": "/var/lib/config-data/default/01-global-custom.conf",
+        "dest": "/etc/watcher/watcher.conf.d/01-global-custom.conf",
+        "owner": "watcher",
+        "perm": "0600",
+        "optional": true
+      },
+      {
+        "source": "/var/lib/config-data/default/02-service-custom.conf",
+        "dest": "/etc/watcher/watcher.conf.d/02-service-custom.conf",
+        "owner": "watcher",
+        "perm": "0600",
+        "optional": true
+      },
```

Comment:
The comma in this line is breaking json format.

Reply:
thank you